### PR TITLE
build: fix link issues if local libs conflict with third-party libs

### PR DESCRIPTION
**Pull request by zhanghaohit**
* **What is the current behavior?** (You can also link to an open issue here)
#1277

* **What is the new behavior (if this is a feature change)?**
use the static libs shipped in thirdparty

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -86,9 +86,20 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
...
 endif ()
+
+find_package(OpenSSL REQUIRED)
+find_library(BRPC_LIBRARY brpc)
+find_library(PROTOBUF_LIBRARY protobuf)
+find_library(GLOG_LIBRARY glog)
+find_library(GFLAGS_LIBRARY gflags)
+find_library(UNWIND_LIBRARY unwind)
+find_library(LEVELDB_LIBRARY leveldb)
+find_library(Z_LIBRARY z)
```
Comment (aceforeverd) [inline CMakeLists.txt:97]: might be other place as well: 
https://github.com/4paradigm/OpenMLDB/blob/aac8167ca47b47871f5c6902abedeb6a7d294c43/src/CMakeLists.txt#L123

**Comment by aceforeverd** [comment]
test failed ?

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: src/sdk/CMakeLists.txt [hunk 2]
```diff
@@ -22,34 +22,34 @@ if(TESTING_ENABLE)
...
     add_executable(mini_cluster_request_batch_bm mini_cluster_request_batch_bm.cc)
-    target_link_libraries(mini_cluster_request_batch_bm mini_cluster_bm_common benchmark_main benchmark gtest ${BIN_LIBS} ${HYBRIDSE_CASE_LIBS})
+    target_link_libraries(mini_cluster_request_batch_bm mini_cluster_bm_common benchmark_main benchmark ${GTEST_LIBRARIES} ${BIN_LIBS} ${HYBRIDSE_CASE_LIBS})
 
     add_executable(mini_cluster_request_bm mini_cluster_request_bm.cc)
-    target_link_libraries(mini_cluster_request_bm mini_cluster_bm_common benchmark_main benchmark gtest ${BIN_LIBS} ${HYBRIDSE_CASE_LIBS})
+    target_link_libraries(mini_cluster_request_bm mini_cluster_bm_common benchmark_main benchmark ${GTEST_LIBRARIES} ${BIN_LIBS} ${HYBRIDSE_CASE_LIBS})
 endif()
 
-set(SDK_LIBS openmldb_sdk openmldb_catalog client zk_client schema openmldb_flags openmldb_codec openmldb_proto base hybridse_sdk zookeeper_mt ${VM_LIBS} ${LLVM_LIBS} ${BRPC_LIBS} ${ZETASQL_LIBS})
```
Comment (zhanghaohit) [inline src/sdk/CMakeLists.txt:52]: verify if remove these duplicate static libs is ok

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**